salt/recommendations/init.sls

```diff
@@ -106,3 +119,11 @@ logrotate-recommendations-logs:
         - template: jinja
         - require:
             - recommendations-logs
+
+recommendations-smoke-tests:
+    file.managed:
+        - source: salt://recommendations/config/srv-recommendations-smoke_tests.sh
+        - name: /srv/recommendations/smoke_tests.sh
+        - mode: 755
+        - require:
+            - recommendations-nginx-vhost
```

Comment:
```suggestion
            - recommendations-folder
```

Reply:
To place the file yes, to actually run the tests nginx needs to have been configured. Good to change for me.